### PR TITLE
PMM-6942 Alert rule template edit

### PR DESCRIPTION
**Pull request by tiagomotasantos**
**What this PR does / why we need it**: Adds actions column to alert rule templates list and modal to edit the template

**Which issue(s) this PR fixes**: https://jira.percona.com/browse/PMM-6942

### CODE REVIEW

File: public/app/features/integrated-alerting/components/AlertRuleTemplate/AlertRuleTemplatesTable/AlertRuleTemplatesTable.tsx [hunk 1]
```diff
@@ -47,6 +48,11 @@ export const AlertRuleTemplatesTable = () => {
         accessor: 'created_at',
         width: '10%',
       } as Column,
+      {
+        Header: actionsColumn,
+        accessor: (template: FormattedTemplate) => <AlertRuleTemplateActions template={template} />,
+        width: '10%',
```
Comment (atymchuk) [inline public/app/features/integrated-alerting/components/AlertRuleTemplate/AlertRuleTemplatesTable/AlertRuleTemplatesTable.tsx:54]: nit: relative width makes the action column too wide or too marrow sometimes, depending on the screen size. Is there a way to define a fixed width for the actions column?

**Review by atymchuk — APPROVED** [review]

File: public/app/features/integrated-alerting/components/AlertRuleTemplate/AlertRuleTemplatesTable/AlertRuleTemplatesTable.tsx (hunk 1 above)
Comment (tiagomotasantos) [reply public/app/features/integrated-alerting/components/AlertRuleTemplate/AlertRuleTemplatesTable/AlertRuleTemplatesTable.tsx:54]: That's right!! I tried a fixed value here but it doesn't work (seems like the width of cells is getting calculated afterwards). So I'm investigating how to do it with react-table.

Comment (atymchuk) [reply public/app/features/integrated-alerting/components/AlertRuleTemplate/AlertRuleTemplatesTable/AlertRuleTemplatesTable.tsx:54]: Can we pass something like `maxWidth: 200`?

Comment (tiagomotasantos) [reply public/app/features/integrated-alerting/components/AlertRuleTemplate/AlertRuleTemplatesTable/AlertRuleTemplatesTable.tsx]: We can but doesn't work either when using percentage in the other columns. But setting all the other columns percentage to add up 100% did the trick. So now the actions column width is fixed!

**Review by nicolalamacchia — APPROVED** [review]

**Review by nicolalamacchia — APPROVED** [review]

**Pull request merged**